### PR TITLE
[rust] Include support for IEDriverServer in Selenium Manager

### PR DESCRIPTION
**Pull request by bonigarcia**
### Description
This PR implements the support for IEDriverServer (i.e., the driver required to use Internet Explorer) with Selenium Manager.

### Motivation and Context
This feature is part of the development of Selenium Manager M2.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [X] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: rust/src/iexplorer.rs [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+use crate::manager::{get_minor_version, BrowserManager};
+
+use crate::metadata::{
+    create_driver_metadata, get_driver_version_from_metadata, get_metadata, write_metadata,
+};
+
+const BROWSER_NAME: &str = "iexplorer";
+const DRIVER_NAME: &str = "IEDriverServer";
+const DRIVER_URL: &str = "https://github.com/SeleniumHQ/selenium/releases/";
+const LATEST_RELEASE: &str = "latest";
```
Comment (diemol) [inline rust/src/iexplorer.rs:33]: As discussed offline. We probably need to build a metadata file with this information?

Comment (bonigarcia) [reply rust/src/iexplorer.rs:33]: IMO, yes, it is a good idea to publish the latest version of the published assets. This info can be used for managers (Selenium Manager and maybe others) to discover the latest IEDriverServer version easily. @titusfortner created an issue for this: #11280

**Comment by titusfortner** [comment]
Oof, I thought this had already been merge when I was trying to run tests on Windows. Looks like it needs rebasing from trunk.

**Comment by titusfortner** [comment]
I built this and verified it works

**Pull request merged**